packages/lfrqa/src/aviary/envs/lfrqa/task.py

```diff
@@ -31,13 +31,8 @@ def __init__(
 
     def get_new_env_by_idx(self, idx: int) -> LFRQAPairwiseEvalEnv:
         """Create a new environment instance for the given index."""
-        question = self.data[idx]
-
         return LFRQAPairwiseEvalEnv(
-            qid=question.qid,
-            question=question.question,
-            human_answer=question.answer,
-            gt_doc_ids=question.gt_doc_ids,
+            query=self.data[idx],
```

Comment:
So now we pass the whole LFRQAQuestion object. I see we do this in other envs as well and thought it was a good oportunity to copy that.

Reply:
Sounds good